bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gems/Canopus.java

```diff
@@ -28,111 +25,36 @@
 
     /**
      * Canopus guide probe options.
-     * <br>
-     * Note for validate:<br>
-     * CWFS1 can vignette both CWFS2 and CWFS3<br>
-     * CWFS2 can vignette only CWFS3<br>
-     * CWFS3 does not vignette other probe arms<br>
+     * Vignetting is no longer an issue due to move to NGS2.
      */
     public enum Wfs implements GuideProbe, ValidatableGuideProbe, OffsetValidatingGuideProbe {
```

Comment:
Given the changes, there is now very little that is unique to each of the CWFS probes, so I was able to simplify these dramatically.